ftw/mobilenavigation/browser/resources/navigation.css

```diff
@@ -139,6 +139,15 @@ div.slideNavi.loading {
   min-height: 200px;
   display: block;
 }
+#slider-container {
+  overflow: auto;
+  position: fixed;
+  top:      56px;
+  height:   100%;
+  height:   calc(100% - 56px);
```

Comment:
Due to http://caniuse.com/#search=calc ``` calc ``` has only partial support for IE9.

I would better use a ``` padding-bottom: 56px; ``` on ``` .mobileNavigation.globalnav ```:

![bildschirmfoto 2015-08-24 um 08 43 03](https://cloud.githubusercontent.com/assets/1637820/9434185/48560c24-4a3c-11e5-8386-bb9bb79c0990.png)